helix-term/src/commands/typed.rs

```diff
@@ -2011,20 +1879,22 @@ fn language(
     if args.is_empty() {
         let doc = doc!(cx.editor);
         let language = &doc.language_name().unwrap_or(DEFAULT_LANGUAGE_NAME);
-        cx.editor.set_status(language.to_string());
+        // PERF: `&str` has a fast specialization for `to_string` so deref `&&str`
+        cx.editor.set_status((*language).to_string());
```

Comment:
I would be a little surprised if the compiler doesn't automatically deref the `&&str`. But in any case we're taking the second ref unnecessarily:

```rust
let language = doc.language_name().unwrap_or(DEFAULT_LANGUAGE_NAME);
cx.editor.set_status(language.to_string());
```